wiki/Beatmap_Editor/Compose/en.md

```diff
@@ -1,406 +1,355 @@
 
 **Insert Break Time** button will start/stop "Break Time" in current timestamp and **x/y** is current mouse coordinate.
 
-Centre-Left (Selector)
-----------------------
+### Centre-Left (Selector)
+
+![Hit Notes Selector](BE_HO.jpg "Hit Notes Selector")
 
-<img src="BE_HO.jpg" title="fig:Hit Notes Selector" alt="Hit Notes Selector" width="200" /> **Your construction toolkit, made for DIY.**
+**Your construction toolkit, made for DIY.**
 
 **Please note that in Standard/osu! 's default notes hitsound works different with osu!mania**
 
 - adding clap on a notes will play 'hitclap' **+ hitnormal** sound on hit while in osu!mania : will only play 'hitclap' sound on hit
 
-<table>
-<thead>
-<tr class="header">
-<th><p>Selector</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>Buttons/ Header<br />
-(Keyboard Shortcut)</p></td>
-</tr>
-<tr class="even">
-<td><p>Sampleset</p></td>
-</tr>
-<tr class="odd">
-<td><p>Additions</p></td>
-</tr>
-<tr class="even">
-<td><p>Select<br />
-(1)</p></td>
-</tr>
-<tr class="odd">
-<td><p>Circle<br />
-(2)</p></td>
-</tr>
-<tr class="even">
-<td><p>Slider<br />
-(3)</p></td>
-</tr>
-<tr class="odd">
-<td><p>Spinner<br />
-(4)</p></td>
-</tr>
-</tbody>
-</table>
-
-Centre (Playfield)
-------------------
+#### Selector
+
+| Buttons / Header (Keyboard Shortcut) | Usage | Description |
+| ------------------------------------ | ----- | ----------- |
+| Sampleset | Auto, Normal, Soft, Drum | This ignore the timing section's Sampleset setting , and add the selected one.... Ex : Timing point with **soft 77%**, While this Sampleset set into **Drum** and have **Clap** on the notes, will play default drum-hitnormal 77% + drum-hitclap 77% on hit |
+| Additions | Auto, Normal, Soft, Drum | Additions hitsound means add Clap, Finish, and Whistle with the SELECTED set... Ex : Sampleset **Normal**, Timing point **Soft 77%**, while Additions is set to **Drum** and have **Clap** on the notes, will result of normal-hitnormal 77% + drum-hitclap 77% on hit |
+| Select (1) | **Left-click/drag:** Select/Adjust the notes/sliderpoints location. **Right-click:** Remove notes/sliderpoints. **Ctrl + Click:** Multiple selection.	| Select and modify existing notes. |
+| Circle (2) | **Left/Right-click:** Add/Remove hit circle. | Hit Circle. |
+| Slider (3) | **Left/Right-click:** Start/End Slider. Add/Remove sliderpoints (white square). **Left-Click x2:** New curve section (red square). | Slider. |
+| Spinner (4) | **Left/Right-click:** Start/End Spinner at current timestamp. | Spinner. Make sure Auto can at least score 2,000 Bonus per Spinner (to prevent the infamous spinner spam/ninja spinner). |
+
+### Centre (Playfield)
 
 **Visual representation of mapping based on current timestamp.** Map your design here and your design will appear exactly like you mapped during play. This is WYSIWYG mapping. For text-based (usually for fine-tuning) mapping, you can open the .osu file directly using Notepad.
```

Comment:
change `This is WYSIWYG mapping.` to `This is WYSIWYG (what you see is what you get) mapping.`
also, change `.osu` to `` `.osu` ``